plugins/gatsby-source-cd-airtable/source-nodes.ts

```diff
@@ -0,0 +1,22 @@
+import { AirTableConnection } from './src/airtable-connection'
+import { config as configDotEnv } from 'dotenv'
+import { SourceNodesArgs } from 'gatsby'
+import { transformProject } from './src/transformers'
+import { createNodesFactory } from './src/create-nodes'
+
+export const sourceNodes = async (
+  sourceNodesArgs: SourceNodesArgs
+): Promise<void> => {
```

Comment:
Tady mi chybí nějaký kontext... proč je návratový typ `void`? Očekává se, že předaná `createNode` action má nějaký side effect, který "magicky" někde data uloží/exposne?

Reply:
Je to tak ... `createNode` exposne data do interního Gatsby GraphQL. Gatsby jenom zavolá `sourceNodes` a nečeká, že mu něco vrátí. Vychází z https://www.gatsbyjs.com/docs/node-apis/#sourceNodes